### DISCUSSION TITLE
Tried calling performUpkeep from contract interface at etherscan. What happens now?

### DISCUSSION THREAD

**Discussion by alperenc** [Cyfrin/foundry-full-course-cu#2411] [Q&A]
I was having issues with my chainlink automation, so tried calling performUpkeep myself. Now it seems stuck. Cancelling the transaction via metamask does not work either. Any suggestions?

**Comment by EngrPips** [comment]
The last time I did the `Lottery` section of the course and used `chainlink automation` I was required by the chainlink Automation to have around 40 Link in my subscription before my operation could be successful, How many Link do you have on your subscription?

**Reply by alperenc** [reply]
Oh wow, I thought that might be the issue and I increased it to 15 but did not think as high as 40 tbh. Let me try that.

**Reply by EngrPips** [reply]
Try to increase it to 40 and retry. Let's see if that helps.

**Reply by alperenc** [reply]
I can't do that since the initial transaction is still pending and I can't cancel for some reason. Do I have to wait it out?

**Reply by EngrPips** [reply]
Deploy a new contract and use the subscription, but make sure to increase the Link balance of the subscription.

**Reply by alperenc** [reply]
I can't deploy a new contract either because the transaction is still pending on my metamask and any subsequent transaction is being queued.

**Reply by EngrPips** [reply]
Maybe use another privateKey for your deployment process. I mean, when I ran into this issue, it wasn't this tough for me.

**Reply by EngrPips** [reply]
Also, I think you can manually send another transaction using the same nonce of the transaction that is pending, and that will automatically cancel the transaction that is pending if the new transaction gets mined.

**Reply by alperenc** [reply]
I waited out, that's not an issue anymore. Back to the original issue, I funded the chainlink automation with 45 LINKs but I don't even see anything pending. The contract returns true for checkUpkeep.

**Reply by EngrPips** [reply]
hmm 🤔, I am not sure what you mean by `I don't even see anything pending`

**Reply by alperenc** [reply]
At around 13:55 in Patrick's video (40/43), his automation shows the transaction as pending due to low balance. I'm not even getting that. I don't think the nodes are even seeing my checkUpkeep being switched to true.

My Raffle contract's address on Sepolia is 0xb4F04F096CB3d9f12B79EfAfd9f0f174EB97E65f, just in case you want to check. I don't know if that helps.

**Reply by EngrPips** [reply]
are you using vrf2.5?

**Comment by Coddium** [comment]
Did you find out what the problem was?

**Comment by EngrPips** [comment]
🤔
